tests/TestCaseBase/Mock/PaymentHandler/PaymentHandlerMock.php

```diff
@@ -23,4 +26,17 @@ protected function getDefaultAuthorizationMethod(): string
     {
         return AbstractRequestParameterBuilder::REQUEST_ACTION_AUTHORIZE;
     }
+
+    protected function createPaymentException(string $orderTransactionId, string $errorMessage, ?\Throwable $e): \Throwable
+    {
+        if (class_exists(PaymentException::class)) {
+            return PaymentException::asyncProcessInterrupted($orderTransactionId, $errorMessage, $e);
```

Comment:
A synchronous exception is thrown in the line below, so the synchronous one should also be used here, right?

Reply:
missed that. thank you for keeping eyes on that.